### ISSUE TITLE
Data Structures to be implemented

### ISSUE THREAD

**Issue by ManrajGrover** [ManrajGrover/algorithms-js#5]
1. Segment Trees
2. Binary Search Tree
3. Graphs

**Comment by itsthecheat** [comment]
@manrajgrover would love to work on this

**Comment by manrajgrover** [comment]
@itsthecheat Yes, please go ahead! Don't forget to write test cases and maintain code quality.

Also, let me know which data structure you're beginning with.

**Comment by willdarnell** [comment]
was this binary search tree ever completed? if not, I would like to do it.

**Comment by anamika2004** [comment]
I want to contribute to this too, are any of the binary search tree or segment tree incomplete? Let me know.

**Comment by robotjellyzone** [comment]
hi @manrajgrover if binary search tree is not yet completed then i want to take this task !!

**Comment by tush38** [comment]
@manrajgrover  can i do the binary search tree implementation?

**Comment by dragosv99** [comment]
hi @manrajgrover  I would like to work on the segment tree. Can I start this task?

**Comment by RodolfoBrian** [comment]
@manrajgrover  I would like to contribute with Binary Search Tree,  if any let me know.

**Comment by Gogotchuri** [comment]
Hello, I wonder if this repository is still maintained? and can I grab those issues?

**Comment by Debanjan10** [comment]
hi  @manrajgrover  I want to contribute to the Binary search tree .

**Comment by VISHESH0932** [comment]
Hey @manrajgrover , i am proficient in Data Structures and Algorithms , would love to work on this issue.
